tfmigrate/state_action.go

```diff
@@ -45,6 +45,14 @@ func NewStateActionFromString(cmdStr string) (StateAction, error) {
 		dst := args[2]
 		action = NewStateMvAction(src, dst)
 
+	case "replace-provider":
```

Comment:
Could you add the replace-provider case to the above godoc comments and unit tests?